android/java/proguard.flags

```diff
@@ -13,6 +13,10 @@
     *** createViewProvider(...);
 }
 
+-keep class org.chromium.chrome.browser.download.DownloadMessageUiControllerImpl {
```

Comment:
This rule should be required for tests only, why we need it in the main proguard file?

Reply:
My mistake. I'll remove it here and will add in tests